### PR TITLE
Docker works

### PR DESCRIPTION
**Pull request by govza**
Make Docker work again.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,8 +1,13 @@
-ALLOWED_HOSTS="*"
-DJANGO_SECRET_KEY = "SomeKey"
-DJANGO_EXECUTION_ENVIRONMENT = "DEVELOPMENT"
-POSTGRES_HOST="db"
-POSTGRES_PORT="5432"
-POSTGRES_USER="postgres"
-POSTGRES_PASSWORD="secret123"
-POSTGRES_DB="mypgsqldb"
+# Django Settings
+ALLOWED_HOSTS = *
```
Comment (al-muammar) [inline .env.example]: It's an environment file. You can't put whitespaces around `=`.
```sh
$ WHATEVER = 25
-bash: WHATEVER: command not found
$ WHATEVER=25
$ echo $WHATEVER
25
```

**Review by al-muammar — COMMENTED** [review]
I would love to accept, but I can't check that it works fine. You might ask @rguliev to review. Maybe he knows docker better.

**Comment by rguliev** [comment]
Ok, I will try to help here.
@govza could you provide a "test plan"? What is a better way to test your PR?

**Comment by govza** [comment]
Check Project is correctly running in docker containers:

Prerequisites:
Installed docker and docker-compose

1. Clone project to machine (if you using docker-toolbox it has to be root directory of Windows ec C:/Users/Public/dev/)
2. On windows run terminal with admin rights (docker terminal, powershell), or use bash on Linux.
3. Check your docker daemon is running 
    `docker-machine status` if it is not "Running" `docker-machine start default`
4. If docker-machine does not exist, create it `docker-machine create default` than run it `docker-machine start default`
5. Go to project directory 
    `cd /path/to/quran-words`
6. Create .env file from .env.example
    `cp .env.example .env`
7. Edit .env file DJANGO_EXECUTION_ENVIRONMENT line, to run project with postgres database
    `DJANGO_EXECUTION_ENVIRONMENT=PRODUCTION`
8. If it's first time you run project on machine, create volume to store persistent data (postgres)
    `docker volume create --name=postgres`
9. Start containers
    `docker-compose up`
10. At first start it should create database for project, so you need to restart the project
    Stop running containers `Ctrl + C`
    Start them again `docker-compose up`
11. Check  containers are running:
    Django at http://127.0.0.1:8000 (for linux) or http://192.168.99.100:8000 (for windows)
12. To access running container. Start new terminal windows, and run
    `docker exec -it web01 sh`
13. Run migrations for django: 
    `python3 manage.py migrate`
14. Check database is seeded by accessing adminer on http://127.0.0.1:8080 (linux) or http://192.168.99.100:8080 (for windows)
15. Login Using: System: PostgreSQL, Server: db, User: postgres, Password: secret123, Database: mypgsqldb (information provided in .env file)
16. There should be tables filled with data quran_ayah, quran_surah, etc.

**Comment by rguliev** [comment]
Actually, I failed at 3th step:) Folder is quran-words repo, DockerWorks branch
```
$ cd quran-words
$ git branch DockerWorks
$ docker-machine start default 
Docker machine "default" does not exist. Use "docker-machine ls" to list machines. Use "docker-machine create" to add a new one.
```

**Review by al-muammar — APPROVED** [review]
docker-i-see

**Pull request merged**